README.md

```diff
@@ -1,2 +1,90 @@
-# CIROH-open-source-project-template
-CIROH open source project template 
+# CIROH Open Source Project Template Instructions
+
+1. Start a new project.
+2. [Copy these files into your new project](#installation).
+3. Update the README, replacing the contents below as directed.
+4. Include any libraries, assets, or hard dependencies whose source code will be included in the project's repository in the _Exceptions_ section of the [TERMS](TERMS.md).
+   - If no exceptions are needed, you can remove that section from TERMS.
+5. If you're working with an existing codebase, answer the questions on the [open source checklist](opensource-checklist.md).
+6. Delete these instructions and everything up to the _Project Title_ from the README.
+7. Get to work and make something awesome!
+
+> Keep the README up-to-date! It's often the first thing people see and forms their initial impression.
+
+## Installation
+
+To install all of the template files, run the following script from the root of your project's directory:
+
+```
+bash -c "$(curl -s https://raw.githubusercontent.com/sepehrkrz/CIROH-open-source-project-template/fe745bff03da976202b376b62cdb42d7de2f92de/open_source_template.sh)"
```

Comment:
This needs to point to the sh file in this repository?

Reply:
Yes. It is pointing to the file in my fork which does the job. we can change the hyperlink to the raw file (open_source_template.sh) after merging the PR. (I am not sure if we can get the raw permalink before the file sits in the repository)

Reply:
Following the format found in the NOAA-OWP version, the link should end up at 
```
https://raw.githubusercontent.com/CIROH-UA/CIROH-open-source-project-template/open_source_template.sh
``` 
I suspect the unique ID in the link is the pull request/commit ID.